### ISSUE TITLE
The model converges very slowly in the fine-tuning phase

### ISSUE THREAD

**Issue by li10107** [Yusufii9/USF-MAE#3]
I fine-tuned the model with your released checkpoints on my in-house dataset, which contains about 1300 B-mode ultrasound images, but i found the model converges very slowly. Even though i set the total epoch to 600, the final accuracy (for a binary classification task) was only about 0.63. The learning rate is 0.001, and an cosine schedule was applied. I did not know the reason behind that.
